WordPress/src/main/java/org/wordpress/android/util/QuickStartUtils.kt

```diff
@@ -67,6 +67,7 @@ class QuickStartUtils {
         @JvmOverloads
         fun stylizeQuickStartPrompt(
             resourceProvider: ResourceProvider,
+            activityContext: Context,
```

Comment:
Since we're passing a Context here, I think we could probably replace the `ResourceProvider` usages with it?

Reply:
Fixed in 64ab82e